cloudnatix/README.md

```diff
@@ -0,0 +1,57 @@
+# Agent Check: CloudNatix
+
+## Overview
+
+This check provides the data from [CloudNatix][1].
+
+CloudNatix connects to multiple VM & K8s Clusters, enabling automated capacity, cost & operation optimization using patent-pending Autopilot technology. CloudNatix Insights also gives DevOps teams proactive visibility into potential capacity and availability issues, turbocharging productivity.
+
+The CloudNatix integration provides cost and operational optimization insights to DataDog with an out-of-the-box dashboard to quickly overview how much the clusters cost and the opportunities to reduce the cost.
+
+## Setup
+
+### Installation
+
+To install the CloudNatix check on your cluster:
+
+1. Build a new Docker image of Datadog agent with installing CloudNatix integration check.
+   Please see [here](https://docs.datadoghq.com/agent/guide/use-community-integrations/?tab=docker)
+   for the details of building that.
+2. [Install the Datadog Agent][3] into your Kubernetes cluster with the built Docker image.
+3. [Install CloudNatix][2] into your Kubernetes cluster. CloudNatix clusteragent will
+   automatically configure itself to work with the Datadog integration.
+
+### Validation
+
+Once set up, you will see several metrics.  Check the dashboard page
```

Comment:
This section is usually used to provide a command that users can run to ensure that the integration is properly installed. Is there a relevant command to list here?

Reply:
I've followed other integration and put the command.